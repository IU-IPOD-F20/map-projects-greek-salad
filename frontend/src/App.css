@import "~antd/dist/antd.css";

#components-layout-demo-fixed .logo {
  width: 120px;
  height: 31px;
  background: rgba(255, 255, 255, 0.2);
  margin: 16px 24px 16px 0;
  float: left;
}
.site-layout .site-layout-background {
  background: #fff;
}

.userTypeWrapper {
  display: flex;
  gap: 3em;
  margin: 0 auto;
  justify-content: center;
}
