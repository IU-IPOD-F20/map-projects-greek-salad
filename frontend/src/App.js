import React from "react";
import { Layout, Menu, Typography, Button } from "antd";
import "./App.css";

const { Header, Content, Footer } = Layout;
const { Title } = Typography;

const App = () => {
  return (
    <div className="App">
      <Layout>
        <Header style={{ position: "fixed", zIndex: 1, width: "100%" }}>
          <div className="logo" />
          <Menu theme="dark" mode="horizontal" defaultSelectedKeys={["2"]}>
            <Menu.Item key="1">Teacher</Menu.Item>
            <Menu.Item key="2">Student</Menu.Item>
          </Menu>
        </Header>
        <Content
          className="site-layout"
          style={{
            padding: "0 50px",
            marginTop: 64,
            minHeight: "calc(100vh - 130px)",
          }}
        >
          <div
            className="site-layout-background"
            style={{
              padding: 24,
              marginTop: 32,
              height: "100%",
              minHeight: "calc(100vh - 130px)",
            }}
          >
            {
              <div className="userType">
                <Title style={{ textAlign: "center" }}>Are you...?</Title>
                <div className="userTypeWrapper">
                  <Button type="text">Teacher</Button>
                  <Button type="text">Student</Button>
                </div>
              </div>
            }
          </div>
        </Content>
        <Footer style={{ textAlign: "center" }}>
          Greek salad Â©2020 Created by Salavat Dinmukhametov, Amir Subaev and
          Lera Vertash
        </Footer>
      </Layout>
    </div>
  );
};

export default App;
