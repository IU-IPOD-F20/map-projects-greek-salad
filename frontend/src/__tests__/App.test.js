import React from "react";
import { MemoryRouter } from "react-router";
import App from "../App";
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});

test("initial render app", () => {
  const wrapper = shallow(
    <MemoryRouter initialEntries={["/"]}>
      <App />
    </MemoryRouter>
  );
  expect(wrapper).toMatchSnapshot();
});
