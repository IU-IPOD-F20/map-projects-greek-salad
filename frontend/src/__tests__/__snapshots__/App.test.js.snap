// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initial render app 1`] = `ShallowWrapper {}`;
